src/cli-options.ts

```diff
@@ -10,7 +10,7 @@ export interface CLIOption<T = any> {
   deprecated?: boolean
   description: string
   help?: string
-  parse?: (s: string) => T
+  parse?: (s: string, p?: any) => T
```

Comment:
I think this should be `p?: string[]` since it is parsing strings from the command line.

Reply:
Yes, I originally just took the type from commander.js `program.options` that expects a `fn: (value: string, previous: any) => any` but `any` should actually be `T` looking at the Typescript definition for it.
Made the change.